src/v8/v8.cc

```diff
@@ -15,7 +15,8 @@
 
 #include "include/proxy-wasm/v8.h"
 
-#include <cassert>
+#include <stdint.h>
+#include <cassert
```

Comment:
It looks that you've accidentally removed `>` from `<cassert>`.

Also, there should be an empty line between C includes (`<stdint.h>`) and C++ includes (`<cassert>`, etc.)

Reply:
I realized that we even don't need to include stdint explicitly, and verified Envoy can be built without it.